modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/suggestion/baseTypes.scala

```diff
@@ -0,0 +1,28 @@
+package com.snowplowanalytics.iglu.schemaddl.jsonschema.suggestion
+
+object baseTypes {
+  sealed trait BaseType extends Product with Serializable
+
+  object BaseType {
+    case object Double extends BaseType
+
+    case object Int32 extends BaseType
+
+    case object Int64 extends BaseType
+
+    case class Decimal(precision: Int, scale: Int) extends BaseType
+  }
+
+  sealed trait NullableWrapper
+
+  object NullableWrapper {
+
+    case class NullableValue(`type`: BaseType) extends NullableWrapper
```

Comment:
In schema-ddl, we tend to use the word `value` only when there is a real data value (e.g. `42` or `"hello"`).  If there is no data value, we tend to use the words like `type` or `field`.